src/index.test.js

```diff
@@ -240,7 +231,8 @@ test('reads text from fs if filePath is provided but not text', () => {
   } catch (e) {
     // ignore
   }
-  expect(fsMock.readFileSync).toHaveBeenCalledTimes(1)
+  // Two hits to .eslintignore
```

Comment:
Here's the line that reads these: https://github.com/eslint/eslint/blob/master/lib/cli-engine.js#L742